packages/appstore/src/components/modals/wallet-modal/__tests__/wallet-modal.spec.tsx

```diff
@@ -28,4 +41,23 @@ describe('WalletModal', () => {
         expect(screen.getByText('WalletModalHeader')).toBeInTheDocument();
         expect(screen.getByText('WalletModalBody')).toBeInTheDocument();
     });
+
+    it('Should render cashier modal if is_wallet_modal_visible is true', () => {
```

Comment:
@thisyahlen-deriv The test description doesn't seem to be matching the test itself 👀